projects.html

```diff
@@ -4,6 +4,7 @@
 <head>
 
 	<title>jordan cooper - projects</title>
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```

Comment:
Question for you; in this case we're referencing the file `https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css`, which ultimately is a file with the name `bootstrap.min.css`. What does the `.min` indicate?